Problem in post phase

I am seeing this error on my build @github:

```
'Post job cleanup.
==> Deleting ".ansible_key" file
##[error]ENOENT: no such file or directory, unlink '.ansible_key'
```
The ansible playbook executes succesfully, but every action ends up in failed state because of the problem in post phase.

I have a configuration that specifies a directory, perhaps it is relevant

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -23,7 +23,7 @@ jobs:
         uses: ./
         with:
           playbook: playbook.yml
-          directory: ./
+          directory: test
           key: ${{secrets.SSH_PRIVATE_KEY}}
           inventory: |
             [all]

--- a/main.js
+++ b/main.js
@@ -20,6 +20,7 @@ async function main() {
 
         if (directory) {
             process.chdir(directory)
+            core.saveState("directory", directory)
         }
 
         if (key) {

--- a/post.js
+++ b/post.js
@@ -8,9 +8,13 @@ function rm(file) {
 
 async function main() {
     try {
+        const directory = core.getState("directory")
         const keyFile = core.getState("keyFile")
         const inventoryFile = core.getState("inventoryFile")
         const vaultPasswordFile = core.getState("vaultPasswordFile")
+
+        if (directory)
+            process.chdir(directory)
 
         if (keyFile)
             rm(keyFile)

--- a/test/playbook.yml
+++ b/test/playbook.yml
@@ -1,0 +1,7 @@
+- name: Test Action
+  hosts: all
+  tasks:
+    - name: Copy action.yml
+      copy:
+        src: ../action.yml
+        dest: /tmp/action.yml

